### PR TITLE
Added unified error handler component.

### PR DESCRIPTION
**Pull request by runishkumar**

### CODE REVIEW

File: tests/widgets/History/components/History.test.js [hunk 1]
```diff
@@ -110,18 +112,19 @@ describe('History component', () => {
...
         />,
       );
 
       // Check no charts anywhere
       expect(widget.find('.panel-body.hidden-xs').find(LineChart).length).toBe(0);
       expect(widget.find('.panel-body.hidden-xs').find(BasicTable).length).toBe(0);
-      expect(widget.find('.panel-body.visible-xs-block').find(LineChart).length).toBe(0);
-      expect(widget.find('.panel-body.visible-xs-block').find(BasicTable).length).toBe(0);
+      expect(widget.find('.panel-body.hidden-xs').find(LineChart).length).toBe(0);
+      expect(widget.find('.panel-body.hidden-xs').find(BasicTable).length).toBe(0);
```
Comment (rocketeric) [inline tests/widgets/History/components/History.test.js]: Why this change? Mistake?

Comment (runishkumar) [reply tests/widgets/History/components/History.test.js]: Yup. Fixed.
